src/Symfony/Component/VarExporter/LazyGhostTrait.php

```diff
@@ -223,6 +223,10 @@ public function &__get($name): mixed
 
                 return $accessor['get']($this, $name, null !== $readonlyScope);
             } catch (\Error) {
+                if (preg_match('/^Cannot access uninitialized non-nullable property ([\w\\\\@]+::\$\w+) by reference$/', $e->getMessage(), $matches)) {
```

Comment:
Technically this will always match, so the regexp is useful only to extract the class:$prop from the message. That's OK because we're not on the hotpath I suppose.
ts```suggestion
                if (preg_match('/^Cannot access uninitialized non-nullable property ([^ ]++) by reference$/', $e->getMessage(), $matches)) {
```